src/app/util/attribute-storage-null-handling.h

```diff
@@ -171,6 +173,51 @@ struct NumericAttributeTraits<BitFlags<T>>
     static uint8_t * ToAttributeStoreRepresentation(StorageType & value) { return reinterpret_cast<uint8_t *>(&value); }
 };
 
+template <typename T>
+struct NumericAttributeTraits<BitMask<T>>
```

Comment:
Instead of duplicating this whole thing, can this not inherit from `NumericAttributeTraits<BitFlags<T>>` and then just implement `StorageToWorking`?  The rest of the implementation for `BitFlags` should be correct here, and passing a `BitMask` to `WorkingToStorage` that expects a `BitFlags` should work, right?